integrationTests/vm/testInitializer.go

```diff
@@ -509,6 +509,9 @@ func CreateVMAndBlockchainHookAndDataPool(
 		MapDNSAddresses: map[string]struct{}{
 			string(dnsAddr): {},
 		},
+		MapDNSV2Addresses: map[string]struct{}{
+			string(dnsAddr): {},
```

Comment:
It has been better do have dnsV2Addr different as dnsAddr here and in L601 for a better testing

Reply:
Unit testing is done on the vm-common.